MenuAl/MenuAl/Screens/LoginScreen/LoginVC.swift

```diff
@@ -8,22 +8,77 @@
 import UIKit
 
 class LoginVC: UIViewController {
+    @IBOutlet var emailTextField: UITextField!
+    @IBOutlet var passwordTextField: UITextField!
+    @IBOutlet var forgotPasswordButtonLabel: UILabel!
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        prepareEmailTextField()
+        preparePasswordTextField()
+        prepareForgotPasswordLabel()
     }
 
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    @IBAction func loginButtonClicked(_ sender: Any) {
+        print("Email Adresi: \(emailTextField.text)")
+        print("Şifresi: \(passwordTextField.text)")
+        navigationController?.pushViewController(HomePageVC(), animated: true)
     }
-    */
+    
+    private func prepareEmailTextField() {
+        emailTextField.layer.cornerRadius = 10
+        emailTextField.clipsToBounds = true
+        emailTextField.layer.borderWidth = 1
+        emailTextField.layer.borderColor = UIColor(named: "grayText")?.withAlphaComponent(0.5).cgColor
+        emailTextField.placeholder = "Email giriniz..."
+        
+        let imageContainer = UIView()
+        imageContainer.translatesAutoresizingMaskIntoConstraints = false
+        imageContainer.backgroundColor = .clear
+        
+        let emailLeftImage = UIImageView()
+        emailLeftImage.translatesAutoresizingMaskIntoConstraints = false
+        emailLeftImage.image = UIImage(named: "email_left_icon")
+        
+        imageContainer.addSubview(emailLeftImage)
+        emailLeftImage.leadingAnchor.constraint(equalTo: imageContainer.leadingAnchor, constant: 15).isActive = true
+        emailLeftImage.trailingAnchor.constraint(equalTo: imageContainer.trailingAnchor).isActive = true
+        emailLeftImage.topAnchor.constraint(equalTo: imageContainer.topAnchor).isActive = true
+        emailLeftImage.bottomAnchor.constraint(equalTo: imageContainer.bottomAnchor).isActive = true
+        
+        emailTextField.leftView = imageContainer
+        emailTextField.leftViewMode = .always
+    }
+    
+    private func preparePasswordTextField() {
+        passwordTextField.layer.cornerRadius = 10
+        passwordTextField.clipsToBounds = true
+        passwordTextField.layer.borderWidth = 1
+        passwordTextField.layer.borderColor = UIColor(named: "grayText")?.withAlphaComponent(0.5).cgColor
+        passwordTextField.isSecureTextEntry = true
+        passwordTextField.placeholder = "Şifre giriniz..."
```

Comment:
NSLocalizedString class ini arastir. Bunu kodun icerisinde nerede kullanabiliriz, neden?

Reply:
Yaptığımız bir uygulamada farklı bölgelerdeki kullanıcılar için yerel dile uygun metinler göstermesini sağlıyor. NSLocalizedString, yazdığımız metinleri direk kod içine sabit olarak yazmak yerine çevirilere uygun hale getiriyor.
Bu kullanımı da  passwordTextField.placeholder = "Şifre giriniz..." burda kullanırız çünkü ekrana bir metin yazdırmışız ve NSLocalizedString classını kullanırsak kullanıcılar bu uygulamayı hangi bölgede de kullanılırsa o bölgenin yerel diline göre otomatik çevirir.